tpp/src/main/TokenClient.js

```diff
@@ -195,6 +198,31 @@ export class TokenClient extends Core {
         });
     }
 
+    /**
+     * Creates a createBulkTransferTokenRequestBuilder for a bulk transfer token request.
+     *
+     * @param transfers List of transfers
+     * @param totalAmount Total amount irrespective of currency. Used for redundancy check.
+     * @param source Source account for all transfer
+     * @returns Builder instance
+     */
+    createBulkTransferTokenRequest(
```

Comment:
Why is this in the unauthenticated client? You definitely need to authenticate in order to use it.